includes/api/ApiQueryUsers.php

```diff
@@ -84,6 +84,19 @@ public function execute() {
 		$users = (array)$params['users'];
 		$goodNames = $done = array();
 		$result = $this->getResult();
+
+		// Wikia change - begin
+		// @see PLATFORM-1561
+		$ids = (array)$params['ids'];
+
+		foreach( $ids as $id ) {
+			$u = User::newFromId( $id );
+			if ( $u->isLoggedIn() ) {
```

Comment:
`isLoggedIn` is kind of misleading as despite what it really means it suggests something like 'is the user logged in or not at the moment'. I would use `! isAnon` here.